### PR TITLE
fix to the fix (missing end_chrom)

### PR DESCRIPTION
**Pull request by northwestwitch**
Thanks @dnil for the previous review. Immediately after I made it available for review I've realized that this is a better solution, sorry!

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/links.py [hunk 1]
```diff
@@ -420,7 +420,7 @@ def ensembl_link(variant_obj, build=37):
     """Compose (sv) variant link to ensembl"""
 
     my_end = variant_obj["end"]
-    if variant_obj["chromosome"] != variant_obj.get("end_chrom"):
+    if variant_obj["chromosome"] != variant_obj.get("end_chrom", variant_obj["chromosome"]):
```
Comment (dnil) [inline scout/server/links.py:423]: Eh, sorry, I'm missing something? Are you trying to avoid some implicit None comparison issue or what?

Comment (dnil) [reply scout/server/links.py:423]: Ah ok. Fine. That is probably better - anything would do really, but yes.

**Review by dnil — APPROVED** [review]

File: scout/server/links.py (hunk 1 above)
Comment (dnil) [reply scout/server/links.py:423]: You could also check if variant_obj.get("end_chrom") was something first, which is sort of what we normally do?

Comment (dnil) [reply scout/server/links.py:423]: ..but this works.

Comment (northwestwitch) [reply scout/server/links.py:423]: I don't know, I just assumed that when end_chrom is missing it should be end_chrom = chromosome. This expression is more clear i think, but you are right, the PR would have worked anyway.

The expression should implicitly first check if variant_obj.get("end_chrom") and of it's not then end_chrom = chromosome, no?

Comment (dnil) [reply scout/server/links.py:423]: I think both would have worked really: if end_chrom == None, my_end=pos, if end_chrom != chr, my_end = pos but if end_chrom == chr my_end=end? In a way I guess the first one is then an easier read, but it is fine this way too.

**Pull request merged**